django_material_demo/polls/templates/material/forms/max_vote_count_form.html

```diff
@@ -0,0 +1,42 @@
+{% load material_form material_form_internal l10n %}
+
+{% form layout=layout %}
+    {% part form.has_max_vote_count prefix %}
+    {% with toggle_id=form.has_max_vote_count.id_for_label  %}
+    <script>
+        $(document).on('change', '#{{ toggle_id }}', function(e) {
+            let formSelector = "form :not(input[name=csrfmiddlewaretoken])";
+            Turbolinks.visit(
+                window.location.origin + window.location.pathname
+                + `?${$(formSelector).serialize()}`
+                + "#{{ toggle_id }}",
+                {action: 'replace'}
+            );
+        });
+    </script>
```

Comment:
Pls don't register onchange listener per element. Generalize this script and I expect the behaviour can be reused by adding an attribute (or some attributes) e.g. `data-reload-form` to the checkbox input or any input element.

Baseline is, we don't need to add a `script` tag each time we need to something similar.